### DISCUSSION TITLE
GDPR concerns on deployments – "Running build in San Francisco, …"

### DISCUSSION THREAD

**Discussion by miguderp** [vercel/community#1477] [Help] [answered]
Hello 👋🏻 

Since a few days ago I started noticing a new message in the build logs, such as _"Running build in San Francisco, USA (West) - sfo1"_.

![image](https://user-images.githubusercontent.com/7307687/217499125-49961961-fe79-443c-9e33-2a220a02a20d.png)

This might sound stupid so apologies if I lack knowledge in this matter but for some European clients couldn't this be a GDPR issue to some extend? My serverless function region is set to Paris but I assume this is something else and not related to the region the build is run.

Is there a possibility to change it? Is this a non-issue (I'm fully open to an explanation if that's the case 😊)

**Comment by amyegan** [comment, marked as answer]
Good question, Miguel! No data is stored permanently inside EU regions and cached resources are short-lived.

You can learn more on our Security page in the section regarding [GDPR compliance](https://vercel.com/security#:~:text=Is%20Vercel%20GDPR%20compliant%3F) and in our [Privacy Policy](https://vercel.com/legal/privacy-policy)

**Reply by ruchernchong** [reply]
I do hope that the team will allow us to select a desired location for our builds to run instead of default to SFO.

**Reply by biapar** [reply]
Hi, How to check where are my data? I cannot run a website outside EU.
_Running build in Washington, D.C., USA (East) – iad1 (Hive)_
